django_grpc/serializers/drf.py

```diff
@@ -0,0 +1,74 @@
+try:
+    from google.protobuf.json_format import MessageToDict, ParseDict
+    from rest_framework import serializers
```

Comment:
this adds dependency on DRF which is not ideal

Reply:
so I use DRF with try, to compatible with somebody don't want use DRF